### PR TITLE
Add not loading notice

### PR DESCRIPTION
**Pull request by i-like-robots**
Due to the [ongoing issue][1] with users being unable to read content on amp.ft.com it has been agreed to add a notice to all content with links back to the content on FT.com to help mitigate the problem. This message will always be visible whether the content or barrier loads successfully or not.

Please note that a compatible version of the `o-message` component is not available so I have improvised by copying the required styles from the build service.

Screen Shot 2019-11-05 at 15 22 32

[1]: https://trello.com/c/4Us4Q6Rg/1327-intermittently-missing-content-in-amp

### CODE REVIEW

File: views/partials/notLoadingNotice.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+			<div class="o-message__content ">
+				<p class="o-message__content-main">
+					<span class="o-message__content-highlight">
+						Content not loading?
+					</span>
+					Subscribers can also read <a href="https://www.ft.com/content/{{id}}?referrer=amp-unreadable" rel="nofollow">{{title}}</a> on <a href="https://www.ft.com">ft.com</a>
+				</p>
+				<div class="o-message__actions">
+					<a href="https://www.ft.com/content/{{id}}?referrer=amp-unreadable" rel="nofollow" class="o-message__actions__secondary">Read now</a>
+					<!-- <a href="https://www.ft.com" class="o-message__actions__secondary">Visit the homepage</a> -->
```
Comment (quarterto) [inline views/partials/notLoadingNotice.html]: commented out code

File: views/partials/notLoadingNotice.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<div class="not-loading-notice">
+	<div class="o-message o-message--alert  o-message--neutral" data-o-component="o-message">
+		<div class="o-message__container">
+			<div class="o-message__content ">
+				<p class="o-message__content-main">
+					<span class="o-message__content-highlight">
+						Content not loading?
+					</span>
+					Subscribers can also read <a href="https://www.ft.com/content/{{id}}?referrer=amp-unreadable" rel="nofollow">{{title}}</a> on <a href="https://www.ft.com">ft.com</a>
```
Comment (quarterto) [inline views/partials/notLoadingNotice.html]: would `webUrl` be more appropriate?

Comment (quarterto) [reply views/partials/notLoadingNotice.html]: the `amp-unreadable` is a great idea btw

Comment (i-like-robots) [reply views/partials/notLoadingNotice.html]: I've updated this to use `webUrl` 👍

**Pull request merged**